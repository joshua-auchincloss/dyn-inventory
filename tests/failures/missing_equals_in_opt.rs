// missing '=' between keyword and value
//
//
//
//
//
//
//
//
//
use dyn_inventory::dyn_inventory;

dyn_inventory!(
    Tt: Ss<T> {
        a: usize,
        t: T,
    };
    macro_name my_macro,
);

fn main() {}
