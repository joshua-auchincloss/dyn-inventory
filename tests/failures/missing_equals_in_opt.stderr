error: expected `=`
  --> tests/failures/missing_equals_in_opt.rs:18:16
   |
18 |     macro_name my_macro,
   |                ^^^^^^^^
